community/en-us/development/DS-License.md

```diff
@@ -0,0 +1,42 @@
+##  DolphinScheduler-Contributor-License Policy
```

Comment:
A total of 14 documents were migrated from `docs/development` to `community/development`, but the original documents were not removed

Reply:
> A total of 14 documents were migrated from `docs/development` to `community/development`, but the original documents were not removed

What you said, I have fixed it